### PR TITLE
test: enforce 100% code coverage

### PR DESCRIPTION
**Pull request by Puppo**
## Summary

- enforce 100% statement, branch, function, and line coverage for authored runtime code
- add focused tests for empty strings, null schemas, and code-generation fallback paths
- keep generated schema validation code outside the coverage scope through .c8rc.json
- leave production runtime code unchanged

## Testing

- npm test
- npm run lint

Coverage reports 100% for statements, branches, functions, and lines.

### CODE REVIEW

File: .c8rc.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "all": true,
+  "check-coverage": true,
+  "lines": 100,
+  "functions": 100,
+  "branches": 100,
+  "statements": 100,
```
Comment (climba03003) [inline .c8rc.json]: Use `--100` in the cli instead.

**Review by climba03003 — CHANGES_REQUESTED** [review]

File: .c8rc.json (hunk 1 above)
Comment (Puppo) [reply .c8rc.json]: done @climba03003

**Review by mcollina — APPROVED** [review]
lgtm

**Review by climba03003 — APPROVED** [review]
